package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fusliez-notes",
-  "version": "0.9.0",
+  "version": "0.9.1",
```

Comment:
Would like to add more things for this patch, so would you mind reverting this change.

Reply:
That's ok